### DISCUSSION TITLE
Bot is placing pixels in the completely wrong spots

### DISCUSSION THREAD

**Discussion by anewphilip** [rdeepak2002/reddit-place-script-2022#248] [Q&A]
Whenever I try to get the bot to draw something, it totally messes up the shape. For example, I just tried to print this extremely simple diagonal line at 1986, 970:
![image](https://user-images.githubusercontent.com/95591461/161429675-d7ac536d-9db4-4d23-a434-1f6df1030107.png)

And this was the result (pixel at 1986, 970 is selected in this image). The colors are right but the positions are all wrong!
![image](https://user-images.githubusercontent.com/95591461/161429660-73909391-5616-481d-8b5d-c5ab8b273603.png)
Here is my config json (excluding username and passwords):
```
{
	"image_path": "image.png",
	"image_start_coords": [1986,970],
	"thread_delay": 2,
	"unverified_place_frequency": false,
	"compact_logging": true,
	"workers": {
		"REDACTED": {
			"password": "REDACTED",
			"start_coords": [0, 0]
		},
		"REDACTED": {
			"password": "REDACTED",
			"start_coords": [1, 1]
		},
		"REDACTED": {
			"password": "REDACTED",
			"start_coords": [2, 2]
		},
		"REDACTED": {
			"password": "REDACTED",
			"start_coords": [0, 2]
		},
		"REDACTED": {
			"password": "REDACTED",
			"start_coords": [0, 1]
		},
		"REDACTED": {
			"password": "REDACTED",
			"start_coords": [0, 2]
		},
		"REDACTED": {
			"password": "REDACTED",
			"start_coords": [0, 50]
		}
	}
}

```

**Comment by thoroc** [comment]
Could you open a bug?
